infra/ballista/build/docker-compose.yaml

```diff
@@ -0,0 +1,22 @@
+version: "3.9"
+services:
+  ballista:
+    build: .
+    image: cloudfuse-io/l12n:ballista
+    cap_drop:
+      - ALL
+    read_only: true
+    volumes:
+      - ballista-tmp:/tmp
+    entrypoint:
+      - python3
+      - lambda-handler.py
+    environment:
+      - AWS_ACCESS_KEY_ID=$LAMBDA_ACCESS_KEY_ID
+      - AWS_SECRET_ACCESS_KEY=$LAMBDA_SECRET_ACCESS_KEY
+      - AWS_SESSION_TOKEN=$LAMBDA_SESSION_TOKEN
+      - AWS_DEFAULT_REGION=eu-central-1
```

Comment:
⁉️ should be set to `AWS_DEFAULT_REGION=$L12N_AWS_REGION`, otherwise it doesn't work for other regions (note that for the other lambdas, the `AWS_REGION` is set to dummy values, but it should also probably be $L12N_AWS_REGION to better simulate the lambda environment)
❓ the actual variable that is present by default in lambda is `AWS_LAMBDA`  (see [docs](https://docs.aws.amazon.com/lambda/latest/dg/configuration-envvars.html)). Is it not detected by the Rust SDK if you use that one? If Ballista really requires the region variable to be called `AWS_DEFAULT_REGION` you should configure set it in the lambda module in terraform.

Reply:
Did you meant AWS_REGION? 
👀 I'm gonna check if it work with that but the error on the cli when not configured was indicating "missing AWS_DEFAULT_REGION"

Reply:
> Did you meant AWS_REGION?

sorry for the typo, updated my initial comment

> I'm gonna check if it work with that but the error on the cli when not configured was indicating missing AWS_DEFAULT_REGION

The hell of inconsistencies between SDKs and codebases with regard to AWS env variables... It never ends!!! 😅

Reply:
confirmed got `DataFusionError(ObjectStore(Generic { store: "S3", source: MissingRegion }))` using AWS_REGION

Reply:
confirming that ballista does not read `AWS_REGION` does not answer either of the two remarks above

Reply:
Sorry for not being clearer. It aims at clarifying the second point.
For the first point the change is implemented towards `$L12N_AWS_REGION`